### PR TITLE
Fix av requirements

### PR DESCRIPTION
**Pull request by Tanguyabel**
Python requirements av==9.1.1 were recently added to `requirements-test.xt` but they seem to be needed in the main requirements, so I moved them to `requirements.txt`.
Note: in this PR I also upgrade sort version because of a known issue:https://github.com/home-assistant/core/issues/86892

---

- [x] This PR references an open issue
- [ ] Added a test
- [ ] Updated the documentation
- [ ] Added a screenshot of a successful customer deployment

### CODE REVIEW

**Comment by IyadhKhalfallah** [comment]
For the failing tests, you can refer to `now.constants`. 
You'll find this list of tags for images
`DOCKER_BFF_PLAYGROUND_TAG = X
NOW_PREPROCESSOR_VERSION = X
NOW_ELASTIC_INDEXER_VERSION = X
NOW_AUTOCOMPLETE_VERSION = X`
You need to update their versions (you can upgrade it and change it by the name of your branch)

File: requirements-test.txt [hunk 1]
```diff
@@ -17,4 +17,3 @@ streamlit==1.11.1
 extra-streamlit-components==0.1.55
 click==8.0.0
 frozendict==2.3.4
-av==9.1.1
```
Comment (IyadhKhalfallah) [inline requirements-test.txt:20]: Do we not need this in `requirements-test`?

**Review by IyadhKhalfallah — APPROVED** [review]
LGTM, nice job 🚀 
Small comment, just for clarification

File: requirements-test.txt (hunk 1 above)
Comment (Tanguyabel) [reply requirements-test.txt:20]: good question; are the two requirements supposed to work together or separately?

Comment (IyadhKhalfallah) [reply requirements-test.txt:20]: I see that they work separately, because there is no dependency between one another

Comment (IyadhKhalfallah) [reply requirements-test.txt:20]: Nice! Then it's good to go ✅

**Review by jupyterjazz — APPROVED** [review]
Congrats on the first PR 🎉

**Review by violenil — APPROVED** [review]
LGTM! Great first PR🚀

**Pull request merged**